tools/linter/README.md

```diff
@@ -0,0 +1,33 @@
+# api.yaml linter
+
+This linter lints api.yaml against the Discovery Docs.
+
+Because api.yaml is handwritten, it will inevitably veer away from the Discovery docs
+(the source of truth) or just be inputted incorrectly.
+
+## Running the linter
+The linter is powered by RSpec.
+
+```
+  rspec tools/linter/run.rb
+```
+
+## Adding new tests
+All new tests should be added in `tools/linter/tests.rb`
+New tests for properties should be added in `property_tests`.
+New tests for resources should be added in `resource_tests`
+
+## Adding new api.yamls
```

Comment:
Stray thought, nothing actionable here: We could probably use a clear name for what an `api.yaml` file is instead of just calling them `api.yaml`s all the time. Something along the lines of "API definition"